### DISCUSSION TITLE
Welcome to market-community Discussions!

### DISCUSSION THREAD

**Discussion by henkvancann** [keri-foundation/market-community#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kentbull** [comment]
I am Kent Bull, a KERI developer working at GLEIF on the vLEI team. I am also a KERI content creator with a few tutorials on my blog (kentbull.com) and a few accompanying Youtube videos on KERI. My plan is to build out a complete tutorial series on building a CESR parser and a KERI implementation as well as how to build a mobile holder app, issuer, and verifier app suite using the KERI & ACDC stack.
